lib/main.dart

```diff
@@ -1,24 +1,172 @@
 import 'package:flutter/material.dart';
```

Comment:
You should do it in templates folder, not just random file

Reply:
Some extension in my VSCode breaks the rendering of files in */lib/templates/*. Therefore I usually move files around to make Intellisense and other functionality available wile I work.

I understand that in real projects it is not possible to just move files around at your will, but for labs I think it is justifiable in my case. I am working on fixing this problem.